src/pages/__tests__/Calculator.test.tsx

```diff
@@ -10,4 +10,21 @@ describe('calculator page', () => {
 
     expect(getByText(/constructed a calculator that lets/i)).toBeInTheDocument()
   })
+
+  it('activity details section reveals after location is selected', () => {
+    const { getByText, queryByText, getByRole } = render(<Calculator />, {
+      wrapper: AllProviders,
+    })
+
+    expect(getByText(/constructed a calculator that lets/i)).toBeInTheDocument()
+    const interactionTypeLabelQuery = /Is this a single activity or an ongoing relationship/i
+    expect(queryByText(interactionTypeLabelQuery)).not.toBeInTheDocument()
+    const topLocationBox = getByRole('combobox', { name: /Select location/i })
+    fireEvent.change(topLocationBox, {
+      target: { value: 'US_06' }, // California
+    })
+    const californiaOption = getByText(/California/i) as HTMLOptionElement
```

Comment:
Can we use this `californiaOption` value to get the select target in the statement above?

Reply:
Oh smart move. Yeah that works. Done.